src/libraries/System.Runtime/tests/TrimmingTests/TypeBuilderComDisabled.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using System.Reflection;
+using System.Reflection.Emit;
+
+AssemblyBuilder assembly = AssemblyBuilder.DefineDynamicAssembly(new AssemblyName("GeneratedAssembly"), AssemblyBuilderAccess.Run);
+ModuleBuilder module = assembly.DefineDynamicModule("GeneratedModule");
+TypeBuilder genericType = module.DefineType("GeneratedType");
+genericType.DefineField("_int", typeof(int), FieldAttributes.Private);
+genericType.DefineProperty("Prop", PropertyAttributes.None, typeof(string), null);
+
+Type generatedType = genericType.CreateType();
```

Comment:
Can we assert something simple about the Type? Just to ensure that it works. For example, maybe create an instance of the Type, call ToString(), and ensure the string is correct.